### PR TITLE
fix: support hoisted packages

### LINKED ISSUE #270: Not working with hoisted packages
I was having a problem with my `babel-plugin-macros` branch, one of the packages of my monorepo didn't compile with the error:

    Error: 'default' is not exported by ../../node_modules/object-assign/index.js

after a bit of tinkering I found out that the problem is this line:

https://github.com/developit/microbundle/blob/master/src/index.js#L359

basically, this line tells `rollup-plugin-commonjs` to include the local `node_modules` folder of the current package, but it will not include any hoisted packages (stored at the root `node_modules` folder).

I tried to change that to `**/node_modules/**` but it still doesn't work, for now the only working attempt is:

```
commonjs({
  include: '../../**/node_modules/**',
})
```

which obviously is not optimal.

I'm quite sure my error could be reproduced even without the additional `babel-plugin-macros` but I can't figure out how.

Do you think it's worth addressing this issue already?

### PR DESCRIPTION
**Pull request by FezVrasta**
This aims to fix #270

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -356,7 +356,8 @@ function createConfig(options, entry, format, writeMeta) {
 						browser: options.target !== 'node',
 					}),
 					commonjs({
-						include: 'node_modules/**',
+						// use a regex to make sure to include eventual hoisted packages
+						include: /node_modules/,
```
[the remark this answers is not in the document]
Comment (FezVrasta) [reply src/index.js]: done

**Comment by ForsakenHarmony** [comment]
Have you tested that this works?

**Comment by FezVrasta** [comment]
@ForsakenHarmony yes that's how I discovered I needed it :-)

**Comment by ForsakenHarmony** [comment]
As long as it works on Windows 👀

**Comment by FezVrasta** [comment]
Unfortunately I don't have any Windows machine to test 😞

**Comment by Andarist** [comment]
@ForsakenHarmony it should work, because its normalized under the hood https://github.com/rollup/rollup-pluginutils/blob/fe8b70a981114becb856dfbc0f1099b6a255ce17/src/createFilter.js#L15

**Pull request merged**